recipes/delly/meta.yaml

```diff
@@ -3,21 +3,17 @@ package:
   version: '0.7.6'
 
 source:
-  fn: delly_v0.7.6_linux_x86_64bit
-  url: https://github.com/tobiasrausch/delly/releases/download/v0.7.6/delly_v0.7.6_linux_x86_64bit
-  md5: 148ed6dde55f07491138226e53198a47
+  fn: delly_v0.7.6_CentOS5.4_x86_64bit
+  url: https://github.com/tobiasrausch/delly/releases/download/v0.7.6/delly_v0.7.6_CentOS5.4_x86_64bit
+  md5: 8efd63cb83f8f18428bbacf8b1b1945f
 
 build:
-  number: 1
+  number: 2
```

Comment:
Build number should be 0. You only need to increase it when you update a build that already exists on anaconda.